src/internal_events/tcp.rs

```diff
@@ -46,7 +46,7 @@ impl InternalEvent for TcpConnectionDisconnected {
     }
 
     fn emit_metrics(&self) {
-        counter!("tcp_connections_disconnected", 1);
+        counter!("connections_disconnected_total", 1);
```

Comment:
Just want to double-check that the metrics in here do not need a `"mode" => "tcp"` tag since the `connection_shutdown_total` metric below does.